.gitignore

```diff
@@ -0,0 +1,82 @@
+
+Reciever/Reciever/.vs/Reciever/v16/.suo
```

Comment:
You can also specify a folder name in .gitignore to ignore everything in that folder